trigger/task_jira.go

```diff
@@ -386,9 +386,14 @@ func (j *jiraBuilder) getDescription() (string, error) {
 	tests := []jiraTestFailure{}
 	for _, test := range j.data.Task.LocalTestResults {
 		if test.Status == evergreen.TestFailedStatus {
+			url := test.GetLogURL(evergreen.GetEnvironment(), evergreen.LogViewerHTML)
+			if url == "" {
+				url = test.GetLogURL(evergreen.GetEnvironment(), evergreen.LogViewerParsley)
```

Comment:
If we're going to call `evergreen.GetEnvironment()`, it would be preferable to call it once and store it as a var.

Reply:
done. 
And re a test: It's not easy to test, and it's simple enough that I think it's okay without one.

Reply:
Hmm, it seems like this test would rely on global state from the env because it uses UI settings from `evergreen.GetEnvironment()`... I guess it's alright to not test it given that it relies on global testing env state.

cc @ablack12 since this is another case of `evergreen.GetEnvironment` making changes untestable (there are a lot of other ones, but this is one example).